seirmo/plots/_plot_from_numpy.py

```diff
@@ -0,0 +1,154 @@
+#
+# This file is part of seirmo (https://github.com/SABS-R3-Epidemiology/seirmo/)
+# which is released under the BSD 3-clause license. See accompanying LICENSE.md
+# for copyright notice and full license details.
+#
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+class ConfigurablePlotter:
```

Comment:
Apologies, didn't think we had to do this if it was a 'private' class, this has been rectified now

Reply:
No worries, thanks for adding it. I mention this because the configurableplotter actually strikes me as a fairly public part of the package.

Reply:
@rcw5890 is it therefore an issue if this class is within the '_plot_from_numpy.py' file - I wasn't sure if that leading underscore nominally marked it as private?

Reply:
You are right that the leading underscore on a module name does suggest that the module itself is private and should not be imported by the user. But, we have exposed the ConfigurablePlotter in [seirmo.plots](https://github.com/SABS-R3-Epidemiology/seirmo/blob/AddProspensity/seirmo/plots/__init__.py), so that class is still public.

I'm guessing they were inspired last year by the decisions made in pints. If you look at their [main directory](https://github.com/pints-team/pints/tree/master/pints), all their files without leading underscores are actual modules you would import (e.g., pints.io, pints.noise, etc), while the files with leading underscores would not be imported even though they may contain code for completely public classes and functions.

Reply:
Okay brilliant we will be consistent with that then!